src/Sarif.UnitTests/Visitors/RebaseUriVisitorTests.cs

```diff
@@ -147,8 +147,8 @@ public void RebaseUriVisitor_VisitRun_DoesNotPatchFileDictionaryKeysWhenNotABase
         [Fact]
         public void RebaseUriVisitor_VisitFileData_PatchesParentUri()
         {
-            Uri rootfileUri = new Uri(@"file://C:/src/root/blah.zip#/stuff.doc");
-            Uri childFileUri = new Uri(@"/stuff.doc");
+            Uri rootfileUri = new Uri(@"file://C:/src/root/blah.zip");
+            Uri childFileUri = new Uri(@"/stuff.doc", UriKind.RelativeOrAbsolute);
```

Comment:
>RelativeOrAbsolute [](start = 62, length = 18)

Isn't it just plain relative?

Reply:
Not according to .NET. :)  which says, 'i can't make a determination'. leading slashes are problematic, in implementation in .NET, if not in the URI spec. 

---
In reply to: [248385717](https://github.com/Microsoft/sarif-sdk/pull/1215#discussion_r248385717) [](ancestors = 248385717)